CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "v1.0",
+    "author": "davidcpage",
```

Comment:
Is this how you want your name to appear on the website? "David Page" or "Myrtle: David Page" might be more human readable.

Reply:
Thanks I've updated this.